fe/fe-core/src/main/java/com/starrocks/memory/MemoryUsageTracker.java

```diff
@@ -91,11 +95,16 @@ public static void registerMemoryTracker(String moduleName, MemoryTrackable obje
     }
 
     public static void trackMemory() {
-        trackMemory(REFERENCE);
-        trackMemory(ImmutableMap.of("Connector", GlobalStateMgr.getCurrentState().getConnectorMgr().getMemTrackers()));
+        long totalTracked = trackMemory(REFERENCE);
+        totalTracked += trackMemory(ImmutableMap.of("Connector",
+                GlobalStateMgr.getCurrentState().getConnectorMgr().getMemTrackers()));
+
+        LOG.info("total tracked memory: {}, heap used memory: {}", new ByteSizeValue(totalTracked),
```

Comment:
Add another 2 metrics?
1. RSS of the java process
2. Size of off-heap memory, like direct buffer pool, etc.